api/build.gradle

```diff
@@ -197,12 +198,15 @@ buildscript {    // Configuration for building
     classpath 'org.owasp:dependency-check-gradle:5.1.0'
     classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.23.0'
     classpath 'net.ltgt.gradle:gradle-apt-plugin:0.21'
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
```

Comment:
Nit: can these be alphabetized, or is the order important?  Sometimes default versions and implicit dependencies mean the latter but accidental dependence on ordering can be avoided by adopting a canonical (alphabetical) order. Or at least that was my understanding of how it worked with Maven.

Reply:
Not sure. I thiiiink the order might matter in cases where you have version conflicts if two packages import the same package.

Given that, I don't think I want to reorder what's already existing but I can move the two new imports into a location that is "ordered".

Reply:
that works for now. I need to learn more about gradle.